### ISSUE TITLE
Easier to read drop menus in gui

### ISSUE THREAD

**Issue by Dad3** [ImpactDevelopment/ImpactIssues#1801]
**Is your feature request related to a problem? Please describe.**
I am frustrated that the sub menus inside the menus are very hard to read because they are too alike to the rest of the menu.
For example here is a column with one of the drop down settings opened:
https://prnt.sc/poxhp7
As you can see, it is very hard to distinct between the settings and the mods.

**Describe the solution you'd like**
I would like to see some sort of outline or a better way to indicate that a drop down menu is opened which will be easier and much cleaner to look at.
I think Flux did a great job of this: http://prntscr.com/poxih8
Basically put an outline or some sort of indentation effect would be nice

**Describe alternatives you've considered**
making it different color completely instead of just white maybe some sort of turquoise because white is very similar to the light grey.

**Comment by Dad3** [comment]
http://prntscr.com/pue5ia the dropdown menu settings look exactly like the mod itself and its very annoying.
Just click the drop down arrow for "tracers" and you'll know what im talking about

**Comment by Quazar5** [comment]
A rather simple solution could be to give the mod letters a custom color so they'd stand out more.
